chore: document variable flags (#2495)
hashicorp/terraform-cdk: website/docs/cdktf/concepts/variables-and-outputs.mdx

});
```

### Passing Input Variables to CDKTF

> You can pass input variables e.g. with the name `imageId` to CDKTF in the following ways:

Reviewer: ```suggestion
You can pass input variables to `cdktf diff`, `cdktf deploy`, and `cdktf destroy` in the following ways:
```

Reviewer: I think the name of the variable (`imageId`) is something that can be inferred – it can also be found in the example above.

- Environment variable: `TF_VAR_imageId=ami-abcde123`

> - CLI argument: `cdktf deploy --var='imageId=ami-abcde123'` or `cdktf diff --var='imageId=ami-abcde123'` or `cdktf destroy --var='imageId=ami-abcde123'`

Reviewer: I'd keep it shorter here and note the commands that support these options in the sentence above.
```suggestion
- CLI argument: `cdktf deploy --var='imageId=ami-abcde123'`
```

> - Auto-Loaded Configuration files: See [Terraform documentation](https://www.terraform.io/docs/configuration/variables.html#variable-definitions-tfvars-files) for details.

Reviewer: ```suggestion
- Auto-loaded configuration files: Refer to the [Terraform documentation](https://www.terraform.io/docs/configuration/variables.html#variable-definitions-tfvars-files) for details.
```
